lhotse/cut.py

```diff
@@ -42,56 +42,68 @@ class Cut:
     """
     id: str
 
-    # Begin and duration are needed to specify which chunk of features to load.
+    # Begin and duration are needed to specify which chunk of features/recording to load.
     start: Seconds
     duration: Seconds
 
+    # Supervisions that will be used as targets for model training later on. They don't have to cover the whole
+    # cut duration. They also might overlap.
+    supervisions: List[SupervisionSegment]
+
     # The features can span longer than the actual cut - the Features object "knows" its start and end time
     # within the underlying recording. We can expect the interval [begin, begin + duration] to be a subset of the
     # interval represented in features.
-    features: Features
+    features: Features = None
 
-    # Supervisions that will be used as targets for model training later on. They don't have to cover the whole
-    # cut duration. They also might overlap.
-    supervisions: List[SupervisionSegment]
+    # For the cases that the model was trained by raw audio instead of features
+    recording: Recording = None
```

Comment:
```suggestion
    recording: Optional[Recording] = None
```

Reply:
I'm wondering if it makes sense for a Cut to have neither Recording nor Features; probably not? But I think I'm ok with not enforcing it for now, we can add it to the list of things to validate when working on #13